kube/match/match-configmap.yaml

```diff
@@ -0,0 +1,20 @@
+# Config Maps store the contents of files in a way kube can access
+# Unless they're in the image, you can't get local files into the pods
+# Created with `kubectl create configmap match-db-config --from-file ./match-db/init.sql -o=yaml`
```

Comment:
Part of me thinks we should put this in a deploy script, rather than committing it?

Reply:
I think this makes sense, as if someone (i.e we regenerate) the SQL file, we can still deploy without updating this file?